data/data_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package data
+
+import (
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+
+	"testing"
```

Comment:
This should be above the other two packages.  stdlib packages first, then external packages, and internal packages each separated by a blank line (to help import sorting).